app/helpers/application_helper.rb

```diff
@@ -46,6 +46,37 @@ def multiple_with_filter?
     params.key?(:search)
   end
 
+  # this helper should be used to print date time in absolute form
+  # it will also define a title with relative time information
+  # it supports only two formats :short and :long
+  # example of long is February 12, 2021 17:13
+  # example of short is Aug 31, 12:52
+  def foreman_absolute_date_time(time, format = :short)
+    content_tag :span, :title => foreman_relative_date_time_value(time) do
+      foreman_absolute_date_time_value(time, format)
+    end
+  end
+
+  # this helper should be used to print date time in relative form, e.g. "10 days ago",
+  # it will also define a title with absolute time information
+  def foreman_relative_date_time(time)
+    content_tag :span, :title => foreman_absolute_date_time_value(time, :long) do
+      foreman_relative_date_time_value(time)
+    end
+  end
+
+  def foreman_absolute_date_time_value(time, format = :short)
+    if format == :short
+      l(time, :format => "%b %d, %H:%M")
```

Comment:
Is there no localized short format?

Reply:
not the one we wanted, that would result in "08 Sep 14:42", the difference is in comma and order

```
Sep 08, 14:42                      # this format
08 Sep 14:42                       # short format
```

Reply:
```08 Sep 14:42``` looks like a EU localized format and for a NL format I'd expect that, not ```Sep 08, 14:42```.

Reply:
I have not seen using `foreman_` as a prefix for methods. Do we want to introduce this here? What about just `date_time_absolute` and `date_time_relative`?

Reply:
Sry for hi-jacking the thread, will avoid that in future

Reply:
@ekohl ok makes sense, I'll change it to short which should be localized
@iNecas the idea was to indicate this is the foreman recommended way which should be used in plugins but if you or anyone else prefers, I can remove the prefix